libs/qlik/src/dashboard-page/stories/vl-qlik-dashboard-page.stories.ts

```diff
@@ -0,0 +1,82 @@
+import { Meta } from '@storybook/web-components';
+import { qlikDashboardPageArgs, qlikDashboardPageArgTypes } from './vl-qlik-dashboard-page.stories-arg';
+import { story, storyArgs, storyArgTypes } from '@domg-wc/common-storybook';
+import '../vl-qlik-dashboard-page.component';
+import qlikDashboardDoc from './vl-qlik-dashboard-page.stories-doc.mdx';
+import filters from '../../utils/stories/demo-filters.json'; // TODO should have an assert
+import viz_without_views from '../../utils/stories/demo-visuals-without-views.json'; // TODO should have an assert
+import viz_with_views from '../../utils/stories/demo-visuals-with-views.json'; // TODO should have an assert
+
+export default {
+    title: 'qlik/qlik-dashboard-page',
+    args: storyArgs(qlikDashboardPageArgs),
+    argTypes: storyArgTypes(qlikDashboardPageArgTypes),
+    tags: ['autodocs'],
+    parameters: {
+        docs: {
+            page: qlikDashboardDoc,
+        },
+    },
+} as Meta<typeof qlikDashboardPageArgs>;
+
+const Template = story(
+    qlikDashboardPageArgs,
+    ({ title, appId, url, exportId, filters, views }) => html` <vl-qlik-dashboard-page
+        title="${title}"
+        app-id="${appId}"
+        url="${url}"
+        export-id="${exportId}"
+        .filters="${filters}"
+        .views="${views}"
+    >
+        <span slot="introduction"> Introduction </span>
+    </vl-qlik-dashboard-page>`
+);
+
+const QlikDashboardPageWithoutViews = Template.bind({});
+QlikDashboardPageWithoutViews.storyName = 'vl-qlik-dashboard-page - without views';
+QlikDashboardPageWithoutViews.args = {
+    title: 'Dashboard page without views',
+    appId: '594c04b6-f319-4cb9-962c-24b0f7aa6f5e',
+    url: 'omgevingsloketrapport.omgeving.vlaanderen.be',
+    views: viz_without_views,
+    filters: filters,
+};
+
+const QlikDashboardPageWithViews = Template.bind({});
+QlikDashboardPageWithViews.storyName = 'vl-qlik-dashboard-page - with views';
+QlikDashboardPageWithViews.args = {
+    title: 'Dashboard page with views',
+    appId: '594c04b6-f319-4cb9-962c-24b0f7aa6f5e',
+    url: 'omgevingsloketrapport.omgeving.vlaanderen.be',
+    views: viz_with_views,
+    filters: filters,
+};
+
+const QlikDashboardPageWithoutFilters = Template.bind({});
+QlikDashboardPageWithoutFilters.storyName = 'vl-qlik-dashboard-page - without filters';
+QlikDashboardPageWithoutFilters.args = {
+    title: 'Dashboard page without filters',
+    appId: '594c04b6-f319-4cb9-962c-24b0f7aa6f5e',
+    url: 'omgevingsloketrapport.omgeving.vlaanderen.be',
+    views: viz_with_views,
+};
+
+const QlikDashboardPageWithExportId = Template.bind({});
+QlikDashboardPageWithExportId.storyName = 'vl-qlik-dashboard-page - with export';
+QlikDashboardPageWithExportId.args = {
+    title: 'Dashboard page with exportId',
+    appId: '594c04b6-f319-4cb9-962c-24b0f7aa6f5e',
+    url: 'omgevingsloketrapport.omgeving.vlaanderen.be',
+    views: viz_with_views,
+    filters: filters,
+    exportId: 'cTTaQLd',
+};
+
+export {
```

Comment:
Wij schrijven het export statement voor elke story ipv beneden in de file voor ze allemaal.